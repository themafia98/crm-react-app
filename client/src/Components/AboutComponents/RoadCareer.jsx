import React from 'react';
import PropTypes from 'prop-types';
import './content.scss';

const RoadCareer = ({content}) => {
    return (
        <p className = 'roadCareer__content'>
            {content}
        </p>
    );
};

RoadCareer.propTypes = {
    content: PropTypes.oneOfType([
        PropTypes.string.isRequired,
        PropTypes.array.isRequired
    ])
};

export default RoadCareer;