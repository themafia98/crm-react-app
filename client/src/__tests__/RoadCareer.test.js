import React from 'react';
import renderer from 'react-test-renderer';

import RoadCareer from '../Components/AboutComponents/RoadCareer';

const RoadCareerTest = renderer.create(
    <RoadCareer content = 'Hello test done!' />
);

test('RoadCareer test', () => {

    let treeSnapshotRoadCareer = RoadCareerTest.toJSON();
    expect(treeSnapshotRoadCareer).toMatchSnapshot();
});