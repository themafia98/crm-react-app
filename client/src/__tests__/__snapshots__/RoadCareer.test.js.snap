// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RoadCareer test 1`] = `
<p
  className="roadCareer__content"
>
  Hello test done!
</p>
`;
